src/main/resources/db/migration/common/V1_387__associatedObjectDataExpiryDeletion_audit_activities.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO audit_activity (aua_id, activity_name, activity_description, created_ts, created_by, last_modified_ts, last_modified_by)
+VALUES
+(36, 'Case documents', 'Case has expired', CURRENT_TIMESTAMP, 0, CURRENT_TIMESTAMP, 0),
+(37, 'Transcription documents', 'Transcription documents', CURRENT_TIMESTAMP, 0, CURRENT_TIMESTAMP, 0),
+(38, 'Annotation documents', 'Annotation documents', CURRENT_TIMESTAMP, 0, CURRENT_TIMESTAMP, 0),
+(39, 'Audio files', 'Audio files', CURRENT_TIMESTAMP, 0, CURRENT_TIMESTAMP, 0);
```

Comment:
I think these names need tweaking to give more context, maybe?

```suggestion
(36, 'Case document expired', 'Case document expired', CURRENT_TIMESTAMP, 0, CURRENT_TIMESTAMP, 0),
(37, 'Transcription document expired', 'Transcription document expired', CURRENT_TIMESTAMP, 0, CURRENT_TIMESTAMP, 0),
(38, 'Annotation document expired', 'Annotation document expired', CURRENT_TIMESTAMP, 0, CURRENT_TIMESTAMP, 0),
(39, 'Audio file expired', 'Audio file expired', CURRENT_TIMESTAMP, 0, CURRENT_TIMESTAMP, 0);
```

Reply:
Agreed updated :)